app/assets/stylesheets/admin.scss

```diff
@@ -32,9 +31,34 @@
   }
 }
 
+// Weekly stats
+.week-selector {
+  margin-top: 18px;
+}
+
+// Pod stats
+.segment-selection {
+  select,
+  [type=submit] {
+    margin-top: 10px;
```

Comment:
It avoids the button to touch the selector. The focus here is the mobile view ;)